lib/ansible/module_utils/aws.py

```diff
@@ -0,0 +1,111 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
Yup, the idea was to leave ec2.py, and to put new work (especially boto3 stuff) into aws.py when it scoped outside just EC2.